src/EnergyPlus/CurveManager.cc

```diff
@@ -199,7 +193,8 @@ namespace CurveManager {
             ShowFatalError(state, "CurveValue: Invalid curve passed.");
         }
 
-        switch (state.dataCurveManager->PerfCurve(CurveIndex).InterpolationType) {
+        auto &thisCurve = state.dataCurveManager->PerfCurve(CurveIndex);
+        switch (thisCurve.InterpolationType) {
         case InterpType::EvaluateCurveToLimits: {
             CurveValue = PerformanceCurveObject(state, CurveIndex, Var1, Var2, Var3, Var4, Var5);
             break;
```

Comment:
I think we are doing `break` outside of the `case: {}` scope.

Reply:
In cases like this with one line operations on each case, I'd be happy to just get rid of the braces and have one line plus a break for each case.